src/Symfony/Component/Console/Helper/Table.php

```diff
@@ -413,6 +413,8 @@ private function buildTableRows($rows)
         $rowsCount = count($rows);
         for ($rowKey = 0; $rowKey < $rowsCount; ++$rowKey) {
             $rows = $this->fillNextRows($rows, $rowKey);
+            // Recount rows
+            $rowsCount = count($rows);
```

Comment:
You moved `count($rows)` out of the for-loop-head into the loop. But you will call it on each iteration again. Its basically the same as before, isn't it?

Reply:
Basically yes. This was done to suppress the warning of static analyzers.

Reply:
I was just wondering, what exactly is now better than before :)

Reply:
It's worst, `count()` is called one time more for nothing :)

Reply:
@dunglas are you sure?

Reply:
You're right it's called the same number of times, but the code is more complex for nothing (and  1 extra line is added)

Reply:
Also `count()` is fast enough, I doubt it's ever gonna make a difference